### DISCUSSION TITLE
ChatGPT integration

### DISCUSSION THREAD

**Discussion by Mekacher-Anis** [outline/outline#6433] [Ideas]
Are there any AI (ChatGPT / Dall-E / StableAI) integrations planned?
something similar to notion, where you a get an input, type in a prompt and let the model, refactor, summarize, rephrase stuff, etc...

**Comment by max-mykhailenko** [comment]
It would be great if Outline had a ChatGPT app similar to Notion that supports both cloud and self-hosting. This would be a game changer for me, as transferring context for each project is currently very frustrating. My current process is:

1. Download the Outline folder with nested documents in Markdown format.  
2. Use a custom script to merge all files into a single large Markdown file.
3. Send this file to ChatGPT for each project or thread.

<img width="1127" height="566" alt="image" src="https://github.com/user-attachments/assets/a6021616-e968-420f-bb64-436a11a3ca98" />

**Reply by max-mykhailenko** [reply]
@tommoor What do you think of this idea? There are not many apps available right now, and it could give the product a strong boost.

**Reply by tommoor** [reply]
I don't believe it's possible because the ChatGPT integration requires a fixed URL, you can use the new [MCP](https://docs.getoutline.com/s/guide/doc/mcp-6j9jtENNKL) integration though on ChatGPT Pro and above by adding it as a custom connector
